src/main/java/com/google/cloud/tools/gradle/appengine/core/DownloadCloudSdkTaskProgressListener.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.google.cloud.tools.gradle.appengine.core;
+
+import com.google.cloud.tools.managedcloudsdk.ProgressListener;
+
+class DownloadCloudSdkTaskProgressListener implements ProgressListener {
+  @Override
+  public void start(String message, long totalWork) {}
```

Comment:
Will these be filled to not be no-op?

Reply:
I'm not sure if it's necessary, given how much information the ConsoleListener already gives.